node/src/actors/chain_manager/mod.rs

```diff
@@ -2131,9 +2133,12 @@ fn current_committee_size_requirement(
     decreasing_step: u32,
     last_consolidated_checkpoint: u32,
     current_checkpoint: u32,
+    last_checkpoint_signed_by_bootstrap: u32,
 ) -> u32 {
-    // If the last consolidated superblock is 0, return the default committee size
-    if last_consolidated_checkpoint == 0 {
+    // If the last consolidated superblock or the current checkpoint is below last_checkpoint_signed_by_bootstrap, return the default committee size
+    if current_checkpoint <= last_checkpoint_signed_by_bootstrap
+        || last_consolidated_checkpoint <= last_checkpoint_signed_by_bootstrap
```

Comment:
If we assume that `last_consolidated_checkpoint <= current_checkpoint`, this can be simplified to

```suggestion
    if last_consolidated_checkpoint <= last_checkpoint_signed_by_bootstrap
```

Reply:
But issue #1526 shows that that assumption is not always true, for some unknown reason.

Reply:
I have not being able to sync with the new testnet from master so I dont think I will be able to reproduce this error. What do we think about it?